prow/cmd/build/main.go

```diff
@@ -190,7 +196,17 @@ func main() {
 		go runServer(o.cert, o.privateKey)
 	}
 
-	controller := newController(kc, pjc, pjif.Prow().V1().ProwJobs(), buildConfigs, o.totURL, configAgent.Config, kube.RateLimiter(controllerName))
+	// TODO remove the feature flag and use directly the config value
+	allowCancellations := false
```

Comment:
Please use the `var allowCancellations bool` pattern for default values

Reply:
fixed